docker/README.md

```diff
@@ -0,0 +1,63 @@
+# 🪐 Docker Image and Container for Big Data Crawler Server
+
+## 🏃 Steps to build the docker image of the Big Data Crawler Server
+
+#### 1. Update (if required) the following version in the `Dockerfile` with a compatible Big Data Crawler Server version.
+
+| Variable                        | Description |
+| ------------------------------- | ------------|
+| big_data_crawler_server_version | The version of the Big Data Crawler Server release to be included in the docker image|
+
+```    
+# Set the following argument with a compatible Big Data Crawler Server version
+ARG big_data_crawler_server_version=1.0.0-SNAPSHOT
+```
+
+#### 2. Open a terminal and change the current working directory to `big-data-crawler-server/docker`.
+
+#### 3. Build the docker image as follows.
+
+```
+docker image build -t nasapds/big-data-crawler-server .
```

Comment:
In documentation (like READMEs), I like to see full arguments always: `-t` → `--tag` … but that's just my personal preference 😇

Reply:
Agreed! Updated the READMEs in all 3 related pull requests.